data/known_rpm_repositories.yml

```diff
@@ -6513,6 +6513,18 @@ rule_data:
     - "rhel-7-workstation-supplementary-debug-rpms"
     - "rhel-7-workstation-supplementary-rpms"
     - "rhel-7-workstation-supplementary-source-rpms"
+    - "rhel-8-appstream"
```

Comment:
Let's drop all the entries without the `-rpms` suffix, see https://github.com/release-engineering/rhtap-ec-policy/pull/86

Reply:
done :)